source/comp/markv_codec.cpp

```diff
@@ -49,6 +49,7 @@
 #include "markv_model.h"
 #include "opcode.h"
```

Comment:
For another time:  It seems these should be source/opcode.h and source/operand.h etc...

Reply:
Yea, there are a bunch of includes which should be fixed. It's on my list, heh.